src/lib.rs

```diff
@@ -19,47 +19,68 @@ fn take_guess() -> char {
         .expect("More than one character was guessed.");
 }
 
-fn run_game_loop(config: &mut Config, input: char) -> GameState {
-    if !config.secret_word.contains(input) {
-        return GameState::LOST;
+struct PlayerState {
+    wrong_guesses: u32,
+}
+
+fn run_game_loop(config: &mut Config, guess: char, player_state: &mut PlayerState) -> GameState {
+    if !config.secret_word.contains(guess) {
+        player_state.wrong_guesses += 1;
+
+        if player_state.wrong_guesses > config.lives {
+            return GameState::LOST;
```

Comment:
Should this be:
```suggestion
        if player_state.wrong_guesses >= config.lives {
            return GameState::LOST;
```

Reply:
lets commit this and see if I get any challenges while merging further down the line